x-pack/plugins/session_view/public/components/session_view_search_bar/index.tsx

```diff
@@ -52,9 +53,20 @@ export const SessionViewSearchBar = ({
 
   const showPagination = !!searchResults?.length;
 
+  const translatePlaceholder = i18n.translate(
```

Comment:
perf improvement: 

declare `translatePlaceholder` with the `placeholder` object flag already:

```js
const translatePlaceholder = {
   placeholder:  i18n.translate(
    'xpack.sessionView.searchBar.searchBarKeyPlaceholder',
    {
      defaultMessage: 'Find...',
    }
  )
}
```

and move it to outside the component, to avoid a new object being created at every render

then use it on the `EuiSearch` bar like that:

```js
<EuiSearchBar
        query={searchQuery}
        onChange={onSearch}
        box={translatePlaceholder}
/>
```